### PR TITLE
SSCS-11392 Court Specific Calendars

### PR DESCRIPTION
**Pull request by markbrooks725**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/SSCS-11392

### Change description ###
Court Specific Calendars for dueDate and delayUntil calculations.

**Does this PR introduce a breaking change?** (check one with "x")
```
[ ] Yes
[X] No
```

### CODE REVIEW

File: src/main/resources/sscs-non-working-days-england-and-wales.json [hunk 1]
```diff
@@ -0,0 +1,45 @@
+{
+  "events": [
+    {
+      "title": "New Year’s Day",
+      "date": "2023-01-02",
```
Comment (DGordonV1) [inline src/main/resources/sscs-non-working-days-england-and-wales.json:5]: Is there a dependency on updating these values or is it more a default file for dmn tables to pull in?

**Review by DGordonV1 — APPROVED** [review]

**Review by DGordonV1 — COMMENTED** [review]
Some checkstyle issues to sort out before approval.

**Review by mrmsali — APPROVED** [review]
Looks good to me.

File: src/main/resources/sscs-non-working-days-england-and-wales.json (hunk 1 above)
Comment (markbrooks725) [reply src/main/resources/sscs-non-working-days-england-and-wales.json:5]: These files will need to be updated annually, I have brought this up with the BA. If the dates are always going to be standard bank holidays plus privilege days we could use the standard calendar for bank holidays with a custom calendar for privilege days.

**Pull request merged**